packages/debugger/test/service.spec.ts

```diff
@@ -96,7 +96,7 @@ describe('DebuggerService', () => {
       path: UUID.uuid4()
     });
     await connection.changeKernel({ name: 'python3' });
-    session = new Debugger.Session({ connection });
+    session = new Debugger.Session({ connection, config });
     config = new Debugger.Config();
```

Comment:
Should those two lines be inverted?

Reply:
Right, done in https://github.com/jupyterlab/jupyterlab/pull/13601/commits/f3074a505a53283f77116b407ada1b1806e2b08d